app/helpers/browser_helper.rb

```diff
@@ -14,15 +14,12 @@ def unsupported_browser?
       # Older versions behind last ESR FF
       return true if browser.firefox? && version < 101
 
-      # Chrome versions older than a year
-      return true if browser.chrome? && version < 109
+      # Chrome/Edge based versions older than a year
+      return true if browser.chromium_based? && version < 109
 
       # Older version of safari
       return true if browser.safari? && version < 16
 
-      # Older version of EDGE
-      return true if browser.edge? && version < 109
```

Comment:
Removal of this line will result in us no longer showing the warning for the non-chromium based edge versions

Reply:
Restore and update the comments